api/lib.go

```diff
@@ -95,11 +97,12 @@ func Handle(cache Cache, code_id []byte, params []byte, msg []byte, store KVStor
 	db := buildDB(store)
 	a := buildAPI(api)
 	q := buildQuerier(querier)
-	var gasUsed u64
+	var gasUsed = u64(0)
```

Comment:
These are the exact same meaning, and the older version is more common in Go, but fine to change it like this... makes it clearer for a Rust/C dev.

Reply:
ah, I wasn't sure if it was or wasn't so i wanted to make it explicit :D
I'll change it back

Reply:
fixed